pac4j-saml/pom.xml

```diff
@@ -191,6 +191,16 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I don't think this is necessary.

Reply:
build failed locally without this dependency. There's a dependency on version 4.4 in opensaml 3.4 while 4.3 is being asserted by one of the deps (having a hard time tracking down which one).

Reply:
Copy that

Reply:
found it; it's xmlsectool. the version I have here is compatible with xmlsectool. Oh, the joys of maven dependency resolution. I've created a property and moved this dep out of the testing block